### DISCUSSION TITLE
Looking for documentation

### DISCUSSION THREAD

**Discussion by cking-spire** [Autodesk/maya-usd#2155] [Q&A]
I there documentation for the mayaUSD python module available anywhere?

Thanks,
Chris King
Spire Animation Studios

**Comment by santosd** [comment]
Hello @CKing-Spire, We have the following documentation on the plugin commands available:
https://github.com/Autodesk/maya-usd/blob/dev/README_DOC.md
I hope that information is helpful.

-David Santos

**Reply by santosd** [reply]
@CKing-Spire I spoke with some of our devs and was told that at this point we don't have much documentation for Maya USD python modules. However, something that could work to get you a little bit more information is to use the python function dir() to list the members available on the object. Cross referencing that with the C++ API documentation should help figure out if the member is a parameter or a function and what the function parameters are.

I have logged a ticket internally for us to add more documentation to our python modules so that the help() function will provide more information.

I hope that helps.

-David Santos
